interfaces/edit_volunteer_cli.py

```diff
@@ -0,0 +1,79 @@
+from models.volunteer import Volunteer
+from models.camp import Camp
+from interfaces.admin_cli import ManageVolunteerMenu
+
+
+###---- Manage Volunteer Menu ----
+class EditVolunteerMenu(ManageVolunteerMenu):
+    def __init__(self):
+        super(EditVolunteerMenu, self).__init__()
+
+    def edit_volunteer_menu(self) -> None:
+        print("""
+         Select an item you'd like to edit:
+         --------
+         1) Firstname
+         2) Lastname
```

Comment:
Firstname should have a space ('First name') and same for last name.

Reply:
Fixed!